StRoot/StFttHitCalibMaker/StFttHitCalibMaker.cxx

```diff
@@ -22,9 +22,11 @@ StMaker("fttHitCalib",name), mHelper(nullptr)
     mHelper = new HitCalibHelper();
 }
 //_____________________________________________________________                                                       
-// StFttHitCalibMaker::~StFttHitCalibMaker()
-// { 
-// }
+StFttHitCalibMaker::~StFttHitCalibMaker()
+{
+  if(mHelper) delete mHelper;
+  mHelper = nullptr;
+}
```

Comment:
[StRoot/StFttHitCalibMaker/HitCalibHelper.h](https://github.com/star-bnl/star-sw/pull/387/files#diff-498e5549bf293b766cf00cd0cfc0c5598244fad0a24e4f01fe1273abad0d0aa9)
Is a bug-fix, but not specifically memory related. The previous algorithm was incorrect. This algorithm does the correct thing, which is to count the number and offset of signals per channel in the detector since that is what we use to calibrate the timing. 

[StFttDb.cxx](https://github.com/star-bnl/star-sw/pull/387/files#diff-6968429fff05ed2873b738c6fd73f1e2192d7cbce9bf0eb387df5ffe98639905)
and
[StFttDb.h](https://github.com/star-bnl/star-sw/pull/387/files#diff-cd9d3c75e76ebd0b314286b71390c8640a9ebd77d8ebfbbd9e01ce754373a406)

fix the fact that this was unusable for calibration (generally without a pre-filled DB) since it was setup to read timing cuts from DB. So a switch was added for when no DB parameters exist, and we want to run in calibration mode with a user-defined timing cut.

Reply:
Thank you for the explanation. So, the title of this PR should be updated to reflect the second change unrelated to the memory fix.